context.go

```diff
@@ -586,8 +586,14 @@ func (c *Context) ListModulePaths(baseDir string) (paths []string, err error) {
 }
 
 type stringAndScope struct {
```

Comment:
stringAndScope was always a bad name, but now it is even less accurate

Reply:
agreed